clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1372,11 +1372,13 @@ public String toEnrichedRst() {
      */
     private void getConfigKeyRst(ConfigKey key, StringBuilder b) {
         b.append("``").append(key.name).append("``").append("\n");
-        for (String docLine : key.documentation.split("\n")) {
-            if (docLine.length() == 0) {
-                continue;
+        if (null != key.documentation) {
```

Comment:
Nit: we typically put `null` on the right hand side in the Kafka codebase. Same below

Reply:
applied